backend/app/interactem/app/operators/__init__.py

```diff
@@ -0,0 +1,82 @@
+import json
+from typing import Any
+import asyncio
+
+from jsonpath_ng import parse
+
+from interactem.app.core.config import settings
+from interactem.app.operators.registry import ContainerRegistry
+from interactem.core.models.operators import Operator
+
+OPERATOR_SPEC_KEY = "interactem.operator.spec"
+
+
+async def _labels(registry: ContainerRegistry, image: str, tag: str) -> dict[str, Any]:
+    manifest = await registry.manifest(image, tag)
+    path = parse("$.config.digest")
+    matches = path.find(manifest)
+    if not matches:
+        raise Exception("Digest not found")
+
+    digest = matches[0].value
```

Comment:
is it possible there are more than 1 match for an image?

Reply:
No, that is just the way the json path interface works ( as it can be used to do multiple matches )